core/src/main/java/com/scalar/db/api/DistributedStorageAdmin.java

```diff
@@ -44,50 +41,6 @@
  */
 public interface DistributedStorageAdmin extends Admin, AutoCloseable {
 
-  /**
-   * Get import table metadata in the ScalarDB format.
-   *
-   * @param namespace namespace name of import table
-   * @param table import table name
-   * @throws IllegalArgumentException if the table does not exist
-   * @throws IllegalStateException if the table does not meet the requirement of ScalarDB table
-   * @throws ExecutionException if the operation fails
-   * @return import table metadata in the ScalarDB format
-   */
-  default TableMetadata getImportTableMetadata(String namespace, String table)
-      throws ExecutionException {
-    return getImportTableMetadata(namespace, table, Collections.emptyMap());
-  }
-
-  /**
-   * Get import table metadata in the ScalarDB format.
-   *
-   * @param namespace namespace name of import table
-   * @param table import table name
-   * @param overrideColumnsType a map of column data type by column name. Only set the column for
-   *     which you want to override the default data type mapping.
-   * @throws IllegalArgumentException if the table does not exist
-   * @throws IllegalStateException if the table does not meet the requirement of ScalarDB table
-   * @throws ExecutionException if the operation fails
-   * @return import table metadata in the ScalarDB format
-   */
-  TableMetadata getImportTableMetadata(
-      String namespace, String table, Map<String, DataType> overrideColumnsType)
-      throws ExecutionException;
-
-  /**
-   * Add a column in the table without updating the metadata table in ScalarDB.
-   *
-   * @param namespace namespace name of import table
-   * @param table import table name
-   * @param columnName name of the column to be added
-   * @param columnType type of the column to be added
-   * @throws IllegalArgumentException if the table does not exist
-   * @throws ExecutionException if the operation fails
-   */
-  void addRawColumnToTable(String namespace, String table, String columnName, DataType columnType)
-      throws ExecutionException;
-
```

Comment:
After the refactoring in `ConsensusCommitAdmin`, these methods are no longer needed.